cdxev/__main__.py

```diff
@@ -661,15 +627,10 @@ def create_set_parser(
         ),
         usage=(
             "cdx-ev set [-h] [--output <file>] [--force] "
-            "(--from-file <file> | <target> --key <key> --value <value>) <input>"
+            "(--from-file <file> | <target> --key <key> --value <value>) input"
```

Comment:
Should be reverted as the `metavar` should be "<input>", as it is a placeholder, which is indicated by the "<>"